docs/src/Microphysics2M.md

```diff
@@ -722,6 +707,28 @@ and the default free parameter values are:
 |------------|---------------------------|
 |``A``       | ``4.7``                   |
 
+### Effective radius - Liu and Hallett, 1997
```

Comment:
Could we move that to the same section as the effective radius computed from the size distribution?